### ISSUE TITLE
Dark theme

### ISSUE THREAD

**Issue by alem0lars-yoroi** [joeyespo/grip#231]
It would be useful to also have some theming options, maily for a dark theme.

**Comment by kba** [comment]
The default output viewed in chromium/chrome with the [Deluminate](https://chrome.google.com/webstore/detail/deluminate/iebboopaeangfpceklajfohhbpkkfiaa) extension works very well for me.

**Comment by alem0lars-yoroi** [comment]
I don't use Chrome, I use uzbl.

**Comment by Atcold** [comment]
This one works perfectly for me https://userstyles.org/styles/37035/github-dark

**Comment by alem0lars-yoroi** [comment]
@Atcold How did you manage to apply that theme to generated html? Did you modify the template?

**Comment by Atcold** [comment]
Add - URLs matching the regexp - `http://localhost:5000/.*`

**Comment by kikijiki** [comment]
That only affects the appearance in the browser, not the contents of the generated html file itself.

**Comment by Atcold** [comment]
@kikijiki, yes, of course. The visualisation engine is going to be GitHub, after all, no?
Right now I have *Grip* previewing the documentation as it will appear in the repo, when I push it.
Both of them have a dark theme applied in the browser.

What is that you're trying to achieve instead?

**Comment by kikijiki** [comment]
I'm using grip to export the file as html to distribute it, it would be cool if the file itself contained the dark style modifications.

**Comment by Atcold** [comment]
Copy and paste the content of the CSS theme file?

**Comment by electrum** [comment]
GitHub now has an official dark mode: https://github.com/settings/appearance

**Comment by aparajita** [comment]
Just posted a PR that implements full GitHub theme support.  #335

If you can't wait for it to merge, uninstall your current grip, then:

```sh
git clone https://github.com/aparajita/grip.git
cd grip
git checkout theming
pip3 install -r requirements.txt

# to run
python3 /path/to/grip/grip/__main__.py
```

You'll probably want to make that last line into an alias to `grip`.

**Comment by unhappychoice** [comment]
mdts comes with 21 built-in themes, each with both light and dark variants! 

It includes Dark themes like Nightfox, Inkstone, Deep Sea, and many others. Just run `npx mdts` and choose your preferred dark theme from the UI.

https://github.com/unhappychoice/mdts

No need to wait 8+ years for dark theme support - it's already there with tons of options!
